apm-sniffer/apm-sdk-plugin/aerospike-plugin/src/main/java/org/apache/skywalking/apm/plugin/aerospike/AerospikeMethodMatch.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.skywalking.apm.plugin.aerospike;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+public enum AerospikeMethodMatch {
+    INSTANCE;
```

Comment:
Why an enum? I think you just need this matcher as static in the `AerospikeClientInstrumentation`.
And are there many other methods in this class? As a result, you have to list all relative methods?

Reply:
[AerospikeClient](https://github.com/aerospike/aerospike-client-java/blob/master/client/src/com/aerospike/client/AerospikeClient.java) class has many methods,but it is possible to remove some methods, I'll remove them.

Reply:
Please remove this enum. This is just a static list.